documentation/docs/examples/timeline.mdx

```diff
@@ -0,0 +1,153 @@
+
+nodes.executeFilter({ assetId: 4222791488928479 });
+const keyframeAssetID = timeline.createKeyframe(new Date('2021-10-06'));
+keyframeAssetID.assignStyledNodeCollection(nodes, { renderGhosted: false, outlineColor: NodeOutlineColor.Orange });
+
+const allPipes = new PropertyFilterNodeCollection(sdk, model);
+allPipes.executeFilter({'PDMS': {'Type': 'PIPE'}});
+const keyframeAllPipes = timeline.createKeyframe(new Date('2021-10-07'));
+keyframeAllPipes.assignStyledNodeCollection(allPipes, { renderGhosted: false, color: [40, 200, 20] });
+
+const stuMechNodes = new PropertyFilterNodeCollection(sdk, model);
+stuMechNodes.executeFilter({'PDMS': {':capDisc': 'MECH', 'Purpose': 'STU' }});
+const keyframeMechNodes = timeline.createKeyframe(new Date('2021-10-08'));
+keyframeMechNodes.assignStyledNodeCollection(stuMechNodes, { renderGhosted: false, color: [80, 100, 120] });
+
+timeline.play(new Date('2021-10-05'), new Date('2021-10-09'), 25000);
+```
+
+### Stop timeline playback
+
+```js runnable
+timeline.stop();
+```
+
+### Pause timeline
+
+```js runnable
+timeline.pause();
+```
+
+### Resume timeline
+
+```js runnable
+timeline.resume();
+```
+
+### Get Keyframe by Date
+`getKeyframeByDate(date: Date)`
+- `date` - Date of the Keyframe which needs retrived.
+
+```jsx
+const keyframe = timeline.getKeyframeByDate(new Date('2021-10-06'));
+```
+
+### Remove Styles & Node in Keyframes
+`unassignStyledNodeCollection(nodeCollection: NodeCollectionBase, nodeAppearance: NodeAppearance)`
+- `nodeCollection` - Node set to be removed from keyframes.
+- `nodeAppearance` - Styles to be removed from keyframes.
+
+```js runnable
+const keyframe = timeline.getKeyframeByDate(new Date('2021-10-06'));
+
+keyframe.unassignStyledNodeCollection(nodes, { renderGhosted: false, color: [150, 200, 120] })
+```
+
+### Timeline Data's while in Play
+While in Play state, the tool emits few data which will be helpful for the users to calculate the progression of the Playback of Timeline.
+Tool provides `TimelineDateUpdateDelegate` event which can be subscribed to received below data.
+- `date` - Virtual Date which can be used to plot progression.
+- `activeKeyframe` - Current Keyframe in the Playback.
+- `startDate` - Keyframe date to start the Playback of Timeline Keyframes.
+- `endDate` - Keyframe date to stop the Playback of Timeline Keyframes.
+
+```jsx
+// import { TimelineDateUpdateDelegate } from '@cognite/reveal/tools';
+
+timeline.subscribe('dateChanged', (date, activeKeyframe, startDate, endDate) => {});
+```
+
+To unsubscribe from `TimelineDateUpdateDelegate` event
+
+```jsx
+timeline.unsubscribe('dateChanged', (date, activeKeyframe, startDate, endDate) => {});
+```
+
+### Removing Keyframe
+#### By Date
+To remove any Keyframe, use `removeKeyframeByDate(date: Date)`
+- `date` - Date of the Keyframe which needs to be removed from the Timeline.
+
+```js runnable
+timeline.removeKeyframeByDate(new Date('2021-10-07'));
+```
```

Comment:
```suggestion
```js
timeline.removeKeyframeByDate(new Date('2021-10-07'));
```
This doesn't work because `timeline` is not in scope. Replace the snippets here with code that's isn't runnable.
```